commit 135d7acdb2c9e6c18ec14f174ea1a84691975df3 (refs/changes/64/2464/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-08-25T10:39:04-07:00 (2 years, 1 month ago)
    
    MANTA-3396 muskie's "mlocate" alias uses incorrect bash alias syntax
    Reviewed by: Tim Kordas <tim.kordas@joyent.com>
    Approved by: Kody A Kantor <kody.kantor@gmail.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index fd34010..98dcc6e 100644
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -7,7 +7,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 set -o xtrace
@@ -209,7 +209,7 @@ manta_setup_muskie_rsyslogd
 
 manta_common_setup_end
 
-#Setup the mlocate alias
-echo "alias=mlocate'/opt/smartdc/muskie/bin/mlocate -f /opt/smartdc/muskie/etc/config.json'" >> $PROFILE
+# Setup the mlocate alias
+echo "alias mlocate='/opt/smartdc/muskie/bin/mlocate -f /opt/smartdc/muskie/etc/config.json'" >> $PROFILE
 
 exit 0
